tests-utils/src/main/scala/tech/beshu/ror/utils/containers/windows/WindowsPseudoContainer.scala

```diff
@@ -0,0 +1,66 @@
+/*
+ *    This file is part of ReadonlyREST.
+ *
+ *    ReadonlyREST is free software: you can redistribute it and/or modify
+ *    it under the terms of the GNU General Public License as published by
+ *    the Free Software Foundation, either version 3 of the License, or
+ *    (at your option) any later version.
+ *
+ *    ReadonlyREST is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *    GNU General Public License for more details.
+ *
+ *    You should have received a copy of the GNU General Public License
+ *    along with ReadonlyREST.  If not, see http://www.gnu.org/licenses/
+ */
+package tech.beshu.ror.utils.containers.windows
+
+import org.testcontainers.containers.GenericContainer
+
+import java.util.concurrent.atomic.AtomicReference
+
+abstract class WindowsPseudoContainer[T <: GenericContainer[T], SERVICE] extends GenericContainer[T]("noop:latest") {
+
+  protected def name: String
+
+  protected def prepare(): SERVICE
+
+  protected def awaitReady(): Unit
+
+  protected def destroy(service: SERVICE): Unit
```

Comment:
Maybe we should introduce an interface for the service instead?
It could have the destroy and getPort methods.

Reply:
Done, new interface `WindowsPseudoContainer.Service`